detector/javadb/javadb.go

```diff
@@ -0,0 +1,100 @@
+//go:build !scanner
+// +build !scanner
+
+package javadb
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/java/jar"
+	"github.com/aquasecurity/trivy-java-db/pkg/db"
```

Comment:
github.com/aquasecurity/trivy-java-db package is directly imported. Please update go.mod with `go mod tidy`.
https://github.com/future-architect/vuls/blob/1071ab759c9feec0e0d99fd30749c323be01eb80/go.mod#L110

Reply:
Fixde at a118ac2243c2fe5a8f1f702d16197b1df5a4e571